app/views/idv/doc_auth/verify.html.erb

```diff
@@ -12,48 +12,51 @@
   <div>
     <%= "#{t('doc_auth.forms.last_name')}: #{flow_session[:pii_from_doc][:last_name]}" %>
   </div>
-  <div>
+  <div class='padding-bottom-1'>
     <%= "#{t('doc_auth.forms.dob')}: #{flow_session[:pii_from_doc][:dob]}" %>
   </div>
-  <hr/>
-  <div class='right'>
-    <%= link_to(t('doc_auth.buttons.change_address'), idv_address_url) %>
-  </div>
-  <div>
-    <%= "#{t('doc_auth.forms.address1')}: #{flow_session[:pii_from_doc][:address1]}" %>
-  </div>
-  <div>
-    <%= "#{t('doc_auth.forms.city')}: #{flow_session[:pii_from_doc][:city]}" %>
-  </div>
-  <div>
-    <%= "#{t('doc_auth.forms.state')}: #{flow_session[:pii_from_doc][:state]}" %>
-  </div>
-  <div>
-    <%= "#{t('doc_auth.forms.zip_code')}: #{flow_session[:pii_from_doc][:zipcode]}" %>
-  </div>
-  <hr/>
-  <div class='right'>
-    <%= button_to(
-      idv_doc_auth_step_path(step: :redo_ssn),
-      method: :put,
-      class: 'usa-button usa-button--unstyled',
-    ) { t('doc_auth.buttons.change_ssn') } %>
-  </div>
-  <div>
-    <%= t('doc_auth.forms.ssn') %>:
-    <%= render(
-          'shared/masked_text',
-          text: SsnFormatter.format(flow_session[:pii_from_doc][:ssn]),
-          masked_text: SsnFormatter.format_masked(flow_session[:pii_from_doc][:ssn]),
-          accessible_masked_text: t(
-            'idv.accessible_labels.masked_ssn',
-            first_number: flow_session[:pii_from_doc][:ssn][0],
-            last_number: flow_session[:pii_from_doc][:ssn][-1],
-          ),
-          toggle_label: t('forms.ssn.show'),
-        ) %>
-  </div>
-
+    <div class='grid-row padding-bottom-1 padding-top-1 address-box'>
```

Comment:
Could we use a [border utility class](https://designsystem.digital.gov/utilities/border/) here instead?

```suggestion
    <div class='grid-row padding-bottom-1 padding-top-1 border-y border-primary-light'>
```